<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMedicalForms</class>
 <widget class="QMainWindow" name="frmMedicalForms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Medical Forms</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="optForms">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Forms</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="optQuestions">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Questions</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpQuestions">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>371</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Questions</string>
    </property>
    <widget class="QLabel" name="lblForms">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Form:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbForm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtQuestion">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblQuestion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Question:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbType">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblType">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSubQuestion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sub-Question:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSubQuestion">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="butAdd2List">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add To List</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="lstQuestions">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>256</width>
       <height>121</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
    </widget>
    <widget class="QCheckBox" name="chkSubQuestion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Sub-Question</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpForm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Form</string>
    </property>
    <widget class="QLabel" name="lblForm">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtForm">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="butClose">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="butEnter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>optForms</tabstop>
  <tabstop>optQuestions</tabstop>
  <tabstop>txtForm</tabstop>
  <tabstop>cmbForm</tabstop>
  <tabstop>txtQuestion</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>txtSubQuestion</tabstop>
  <tabstop>butAdd2List</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
